tools/wasme/cli/pkg/cmd/deploy/options.go

```diff
@@ -94,6 +109,7 @@ func (opts *istioOpts) addToFlags(flags *pflag.FlagSet) {
 	flags.StringToStringVarP(&opts.workload.Labels, "labels", "l", nil, "labels of the deployment or daemonset into which to inject the filter. if not set, will apply to all workloads in the target namespace")
 	flags.StringVarP(&opts.workload.Namespace, "namespace", "n", "default", "namespace of the workload(s) to inject the filter.")
 	flags.StringVarP(&opts.workload.Kind, "workload-type", "t", istio.WorkloadTypeDeployment, "type of workload into which the filter should be injected. possible values are "+strings.Join(SupportedWorkloadTypes, ", "))
+	flags.StringVarP(&opts.patchContext, "patch-context", "", "inbound", "patch context of the filter. possible values are "+strings.Join(SupportedPatchContexts, ", "))
 	flags.StringVarP(&opts.istioNamespace, "istio-namespace", "", "istio-system", "the namespace where the Istio control plane is installed")
```

Comment:
Two minor things - First we can use `StringVar` instead of `StringVarP` if we don't have a short-hand for the flag. (Looks like this was also done for istioNamespace before, so I've fixed that too).
Second - we can use the `PatchContext_Inbound` const from above here as the default value.
```suggestion
	flags.StringVar(&opts.patchContext, "patch-context", PatchContext_Inbound, "patch context of the filter. possible values are "+strings.Join(SupportedPatchContexts, ", "))
	flags.StringVar(&opts.istioNamespace, "istio-namespace", "istio-system", "the namespace where the Istio control plane is installed")
```